cmake/CMakeLists.txt

```diff
@@ -111,6 +111,10 @@ if(onnxruntime_USE_OPENMP)
   endif()
 endif()
 
+# Guarantee that the Eigen code that you are #including is licensed
+# under the MPL2 and possibly more permissive licenses (like BSD).
+string(APPEND CMAKE_CXX_FLAGS " -DEIGEN_MPL2_ONLY")
```

Comment:
would add_definitions() be better?

Reply:
ok. don't have a strong preference. our files already use multiple ways :) fixed.